tdrs-frontend/package.json

```diff
@@ -6,7 +6,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.32",
```

Comment:
@alexsoble Snyk didn't create this in HHS. Should we merge it in and include it one of our PRs?